app/src/test/java/org/astraea/metrics/jmx/MBeanClientTest.java

```diff
@@ -0,0 +1,17 @@
+package org.astraea.metrics.jmx;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.net.MalformedURLException;
+import org.junit.jupiter.api.Test;
+
+class MBeanClientTest {
```

Comment:
It would be better to build a jmx server locally and test all getter by local MBean server (see https://docs.oracle.com/javase/7/docs/api/java/lang/management/ManagementFactory.html#getPlatformMBeanServer())

Reply:
Didn't know that thing exists, I will try it later.

Reply:
記得補上這邊提到的測試喔